tools/editor/io_plugins/editor_font_import_plugin.cpp

```diff
@@ -1592,6 +1592,9 @@ Ref<BitmapFont> EditorFontImportPlugin::generate_font(const Ref<ResourceImportMe
 	int top_space = from->get_option("extra_space/top");
 	int bottom_space = from->get_option("extra_space/bottom");
 	bool enable_filter = from->get_option("advanced/enable_filter");
+	if (from->has_option("advanced/disable_filter")){
```

Comment:
Could you add a comment saying that this is a compatibility check for a deprecated option? This way we could drop it for 3.0 or later if we want.